pkg/kubectl/cmd/config/use_context.go

```diff
@@ -47,6 +47,7 @@ func NewCmdConfigUseContext(out io.Writer, configAccess clientcmd.ConfigAccess)
 	cmd := &cobra.Command{
 		Use:     "use-context CONTEXT_NAME",
 		Short:   i18n.T("Sets the current-context in a kubeconfig file"),
+		Aliases: []string{"use"},
```

Comment:
I don't know if this alias fits with the existing sub-commands which explicitly use `(action)-context`

```
  current-context Displays the current-context
  delete-cluster  Delete the specified cluster from the kubeconfig
  delete-context  Delete the specified context from the kubeconfig
  get-clusters    Display clusters defined in the kubeconfig
  get-contexts    Describe one or many contexts
  rename-context  Renames a context from the kubeconfig file.
  set             Sets an individual value in a kubeconfig file
  set-cluster     Sets a cluster entry in kubeconfig
  set-context     Sets a context entry in kubeconfig
  set-credentials Sets a user entry in kubeconfig
  unset           Unsets an individual value in a kubeconfig file
  use-context     Sets the current-context in a kubeconfig file
  view            Display merged kubeconfig settings or a specified kubeconfig file
```

Should probably get some buy-in from sig-cli UX. Shorter isn't always better.

Reply:
> Shorter isn't always better.

Agreed. I'd prefer to keep it unchanged.

Current aliases in kubectl are only used as shortcuts for various resources, such as `deploy`, `rs`.  We'd better not breaking the rules only for this subcommand.

Reply:
not sure current aliases in kubectl are only used as shortcuts for various resources. but this shortcuts just used for cli. we can not set  ```deploy```,```rs``` in yaml, so maybe we don't have limit for use shortcuts.

Reply:
I'm neutral on this change. Adding an alias `use` for `use-context` is not bad, since we don't have other command prefix with `use`.

@fabianofranz thoughts?

Reply:
We indeed avoid the use of aliases as much as possible. We verified in past that its leads to confusion in the head of users (was it `delete` or `remove`??) much more often than convenience, so we prefer to use suggestions and educate our users to the one single command available that does each thing.

That said, in some very specific areas they are valid and in my opinion this one (`use` specifically) is a good one. Switching contexts is a very common action if you use multiple of them. In OpenShift in past we had it as a top-level command and I remember using `rhc use dev`, `rhc use stg`, `rhc use prod` a lot.

So I'm in favor of `use` → `use-context`, but only this one.

Reply:
great~ at least have became easy for me～.～ thanks you for everyone viewpoint:)  i ‘d like just add use-->use-context.

Reply:
Want to chime in that if we ever get a way to manage multiple kubeconfigs  (https://github.com/kubernetes/kubernetes/issues/38791) then `kubectl config use` seems like the natural way to switch between them.